test/stablecoin_converter.js

```diff
@@ -173,6 +174,45 @@ contract("StablecoinConverter", async (accounts) => {
 
       await truffleAssert.reverts(stablecoinConverter.addToken((await ERC20.new()).address), "Max tokens reached")
     })
+
+    it("Burn 10 OWL on Add Token", async () => {
+      const TokenOWLProxy = artifacts.require("../node_modules/@gnosis.pm/owl-token/build/contracts/TokenOWLProxy")
+      const owlToken = await TokenOWL.new()
+      const owlProxyContract = await TokenOWLProxy.new(owlToken.address)
+      const owlProxy = await TokenOWL.at(owlProxyContract.address)
```

Comment:
Is the proxy needed for the unit test? I would suggest that the owl token be used directly here to make the unit test more concise.

Reply:
I believe that proxy is the only way OWL tokens can be accessed...

Reply:
Yeah, the proxy is needed for the unit tests. 
When I created the owl token, I created it in such a way that it can not be used without the proxy, as the creator is only set in the constructor of the proxy.

At that point in time, when I was writing the owl contracts, I had not such awesome reviewers telling such good principles...